guides/git.md

```diff
@@ -0,0 +1,11 @@
+# Guides for using Git
+
+- config author, email
+- same name for sign off
+- commit writing
+- use location prefix for commit message
```

Comment:
I think @teodutu is right. Generally, you would use the component prefix. Which generally maps to a path. But, in case the path is too large / cumbersome to write, you could just write a component prefix.